prow/plugins/cat/cat.go

```diff
@@ -201,5 +201,15 @@ func handle(gc githubClient, log *logrus.Entry, e *github.GenericCommentEvent, c
 		return gc.CreateComment(org, repo, number, plugins.FormatResponseRaw(e.Body, e.HTMLURL, e.User.Login, resp))
 	}
 
+	var msg string
+	if category != "" {
+		msg = "Bad category. Please see http://thecatapi.com/api/categories/list"
+	} else {
+		msg = "http://thecatapi.com appears to be down"
+	}
+	if err := gc.CreateComment(org, repo, number, plugins.FormatResponseRaw(e.Body, e.HTMLURL, e.User.Login, msg)); err != nil {
+		log.WithError(err).Println("Failed to leave comment")
```

Comment:
s/Println/Error/

This also should be fixed above.

Reply:
Without this we can't filter for errors in the logs with `level=error`.

Reply:
Done